autocompleter/base.py

```diff
@@ -287,21 +288,21 @@ def store(self, delete_old=True):
         norm_terms = self.__class__._get_norm_terms(terms)
         score = self._get_score()
         data = self.get_data()
-        facets = self.get_facets()
+        facet_names = self.get_facets()
 
         # Get all the facet values from the data dict
         facet_dicts = []
-        for facet in facets:
+        for facet_name in facet_names:
             try:
-                facet_dicts.append({"key": facet, "value": data[facet]})
+                facet_dicts.append({"key": facet_name, "value": data[facet_name]})
             except KeyError:
                 continue
 
         old_norm_terms = self.__class__.get_old_norm_terms(obj_id)
-        old_facets = self.__class__.get_old_facets(obj_id)
+        old_facet_dicts = self.__class__.get_old_facet_dicts(obj_id)
 
         norm_terms_updated = norm_terms != old_norm_terms
-        facets_updated = facets != old_facets
+        facets_updated = facet_names != old_facet_dicts
```

Comment:
@anthonyp97 This is the bug you fixed you fixed in #38 - comparing facet_names with facet dicts. Decided to leave it alone for now because fixing it breaks a ton of tests, so I think it warrants it's own PR